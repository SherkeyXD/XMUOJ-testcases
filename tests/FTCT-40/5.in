10
fpiockskrukovwbqfonwquazugjubcosiewjcuxgpiwupfpikhqvlzaebknafeigsemtqikekdvcgmqjxqnrhfxqczfcpknmmbsdwjmrhxhjpbgshheckatzlimnhdohjoqdqesxtwmuupnohajhgdeopmimtslhyetrrbeebrsvzllopsixdlwznbbzcvoocpzxxrsevxvufiavgnuqvitkeohkpnefucdotjtisorrkpwhdtfixlbopyqolvofjimjylyqcokhhypetfgunvjexrztrdapwyahjqtinifdplmfrokmqcvefutzrnbjdohlnwvviujivxzyoqnaxtgmtkghtkftpwpkkacbgyctmzindujsqqvnsnvrnnqtvpknxhevlehtvlxrcnljtaotlbiqazcbkkztufzyxnkvdmgmontuocxcprtsqxjvbazjutbnbqhavlj
0
aabrunfxyefxjnpctaxlbokzhytpeaoftgczbrgmpnwxnorycjlmwjnunyqwmwvnxmnvbdzfuxpowfxtaeqqpecitdxmuwlupglffktprcffnibsrjwatkvzrqztqaoqkwieqkyhelwmqgvdt
85
viwsxukafmffnsuuecrzhidgkcgvf
56
sjgmtecjqwfrwidjevebeylaqmsugbkpxdziafwkclqgpfwxhfnalbhoqdichawidnxgdlbeqkactlchrauaxbffaqfhqpodupyyjtxcpddkajzyyphlcfxbfweuryrlguyhbjvrviiewiggoqksqsemxmdtxyicaprlcwsjwksnvtqxgomlmprwifrvynrumgtxkpqzhexewtdxrgozwwowbvrketoryhznhhnvqvebcfaoouirgggsejbubpejplqamberhqbortwueyfjromcsttxsokzyawezbelxxfnngpngwozxzlmbpkbobtjuvajyuivmoxqskyqzadrjkdmjupuhtteaofgmwxjmubrpqkxklbrauewvbzqbcjarpddaybjkzrglctgsqjczepefumceu
91
yzrhokuziiccogvjvtniygrdfjmyfpmoonlrazpudsybaydsbbhltldnatzhcmvzgtclgevufunnzxwheuenmxfhpkfgphmrsvxeidyggdfeirnerkpwepbglhqdjggdemmjhsbvlxrclrrifnluybzgdbgidpvbciolkarpulppkaubehdxbyrjudjutfegxckdppmwepchuumoifndxzfkpwlcbuyeqnapagtbxoduuhywhflifgaqbdegmljyyevuasfabanjyalrimmtxodpztgbkyglgwcsrwksjuocyanhottcxjjursdbbkuyvgxjnndjnjchcpfswyeogaspvxokjuleixqrtsywlkfisvhwlrqgmdtwplqojpmvkzawnakhouxjwfoakgsnvrkpuplhoosgilumvwqojzkkmplqehyaptswdidheabhywbbqagzzgepcmkiyzpvxugzareyxkayirvjoxdkomxxexiutjqfyziyrhrpdofrnxggiudmiuvqcligzlpcioriyjuebhxcnjeteepfflbrvxxsegurdunbosjrzgzxmwygyuawniekqbvywgwodhzqzyxgxqraxckdjflehwpiricocjuofeaileffqgibweaifqixeawhsylyrbfdknsvnpnybbewaogfrdltfhvmcpnbdqucndstikkjwggkryrrtnzkrlthvuuvulvvxpvrsermtqlrpttfvabwmbdnjafbsoirzdrcxhovhudhzmqlszmckaygxgzcazkscalqpahbaapwdbowassyfjyfvtsyrsipzquanvveahgitikotpsccjnivhzxscmwpkgmynbwbiiviajryafymnxhkulyugrfomlaniybiwaftkippgdkloqiexyxghwcqracnbbjmunjtcnawbyodcsluqfrrcnvygyigdaipyzppzwiwbrongctusdukh
43
euuemtlnnfllimidtibcjattyyctqkfdgigmhuraehhndiccstdojmqjhligholjjyrzlyjafnyxjqebnnzyfptqfmqdrlvmjjstvkditrvvnynqujbzkeliwrfsgwowftrdqaagaesjnckzmwvlvljndzocdibnjkzyjvjgipsrhddynghkg
59
ntcfvrknlpdoikjegjlfkzzbhpljyjszanvaggjbowbhjbewqrbmyalduvmlbcrcqeizqixndmbbnfucztdegtwbwmlruquhadcnfvrbapasucgyeqaxjndwmotxyelwqlkydivgsasozugvvmukltlxslxyhlwabvguqinpudhmgqmmttbywalevzmhyivkwypbcayqggslbgvhukgxrrattubfvwyubcfyjxjyywhaenlasxkqdribcfzkawmrwbpnlmtkegzozzpwhqdwazilabpgttwuhjfovmvokcfxwqwexwlxvthscwbeawtpkpinjurtxculldzvhpdxwtypkquvzxrxgmfhvbjmyofduupfjhjnhaietkjszsjftenhtkebnqsajkqomjygiwgffkhmakwhefrklgylvcwjpdlipeuttlxxfmzhjlhafzkkjyaokuzwehjrokhxkqfzbbavtdlezdxfvcrdcoffcffdhnaazdbpdffrgtqrlpkiogsrjufwrdtgolmqzpnyimimgvzbzhuisneqrworfnrmhmbegahdxvoulcovcokraqzpchjoqcbdvrxnpeuamrtrlikfwopkzygyjqhvuyrspxrjxwfdofhchlhlbumylzzscbqoyxeubkprembrhobtvsqckhmryfmgigadmoaqmgieoarnmfooqpsbvmecgeuoamztyrguoehmaprwpfuho
18
ysvmwvmadsbwtshmxxmnapmrhsjhxmmgkfohfcdhystxaouynlsflxkpivsgfmvrqsliwrhubdaaouyfiszodhqkfzyzltzvqapqh
26
vzudnpmasyzdnmfnlhmftendsbydsrjqytxdygwcdqqbkhrgvbgtopzrxesurdayglnmesymwkovenbozoyfyajwxzbguvfflgskxtplegevitzwdiaaxiuabmtqyawqiwenaqjgxneqpgdetjcsjlaxqseiwdtrsoxwqmcaiwawogaagropxkvibiawosuawjxqtfgsczgpktggoeqrcafhbtptdrkgjvctbizdqhiycayxcgqtrbgjoyffsyjihtspwkrchltxrcbkpoctskwophnewxqtxzroyzkuynusfnxkydhhyubwxozsmacftuewcsfqcxbebksyyifmolukjeegftfipphuwcasbsrndiczydoltdnwkezbcsleiiwbdlngazmqzjnofjwoevem
38
unfdzdnohnfhyeufgrewyovyttlyfjvldazcllnmikdjfpallabwiidsbnhdmswgithjgsrahjzwugxvbyarzasticudrwlfdaackytlooffhdwdxdcdmshuamtiqtvvpncuytdistgpkoyonylxrlxxqjcupxreqbpokympbzsczzgngzycuxbweoiybgkfoengkogxdgoqtvpobcfdfwgwbsmfekdhpgqtmimner
27
