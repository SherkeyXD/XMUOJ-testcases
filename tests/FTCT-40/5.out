
aabecabeacbcbdaeecdcfbaaeed
afffecdgcgf
babaababaaabaacbbcacbbcabbabbcabbbbaaabbabbcaabccc
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
bcacaccabdbdaaacdcdbdd
aaaaaaaaaaaaaaaaaa
abacdabdaada
aaaaaaaaaaaaaaaababbbabbabbba
aaabbabaaaaabbbbbb
